### PR TITLE
refactor: move to /authorization/sources/built_in_database/rules

### PR DESCRIPTION
**Pull request by sstrigler**
Fixes [EMQX-8372](https://emqx.atlassian.net/browse/EMQX-8372)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/` dir
- [x] For internal contributor: there is a jira ticket to track this change
- [x] Coordinate with Dashboard-Team

### CODE REVIEW

**Comment by sstrigler** [comment]
@lafirest could you help with translations please?

File: apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf
Comment (lafirest) [inline apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf:97]: 清除内置数据库中的所有类型（'users' 、'clients' 、'all'）的所有规则

File: changes/v5.0.13-zh.md
Comment (lafirest) [inline changes/v5.0.13-zh.md:13]: 重构 `/authorization/sources/built_in_database/`  接口，将 `rules/` 添加到了其路径中 [#9569](https://github.com/emqx/emqx/pull/9569)。

File: apps/emqx_authz/src/emqx_authz_api_mnesia.erl
Comment (HJianBo) [inline apps/emqx_authz/src/emqx_authz_api_mnesia.erl:73]: 🤔  We should not delete the existing interface directly, which will bring compatibility problems.
IMO, we should mark the existing interfaces as deprecated but still accessible. and add new interfaces under the `rules` path?

Comment (id) [reply apps/emqx_authz/src/emqx_authz_api_mnesia.erl:73]: @HJianBo the background of this task is to make as many breaking changed in our API as possible before the e5.0.0 release.

**Comment by id** [comment]
@sstrigler please rebase on latest master to unblock windows and macos builds

File: apps/emqx_authz/src/emqx_authz_api_mnesia.erl
Comment (HJianBo) [reply apps/emqx_authz/src/emqx_authz_api_mnesia.erl:73]: Got it. thanks for your explanation

**Comment by thalesmg** [comment]
I think the base branch needs to merge or rebase with master, because the way changelogs are constructed has changed: https://github.com/emqx/emqx/pull/9682

**Comment by sstrigler** [comment]
> I think the base branch needs to merge or rebase with master, because the way changelogs are constructed has changed: #9682

My idea is to merge all things into `dev/api-refactor` first and then fix it there before we merge the integration branch.

File: apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf
Comment (thalesmg) [inline apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf:90]: Reminder: needs translation.

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf
Comment (sstrigler) [reply apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf:90]: @lafirest could you help out here please?

Comment (lafirest) [reply apps/emqx_authz/i18n/emqx_authz_api_mnesia_i18n.conf:90]: 删除 `all` 规则

File: changes/v5.0.13-zh.md
Comment (lafirest) [reply changes/v5.0.13-zh.md:21]: 添加新 API 接口 `/bridges_probe` 用于测试创建桥接的参数是否可用。

**Pull request merged**
